extras/src/main/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java

```diff
@@ -193,7 +193,7 @@ private RuntimeTypeAdapterFactory(Class<?> baseType, String typeFieldName, boole
   }
 
   public <R> TypeAdapter<R> create(Gson gson, TypeToken<R> type) {
-    if (type.getRawType() != baseType) {
+    if (!baseType.isAssignableFrom(type.getRawType()))
```

Comment:
Lost opening brace somehow, it does not even compile, otherwise - great and needed fix that does simplify life without negative consequences.

Reply:
Fixed.